copier.yml

```
@@ -46,6 +46,8 @@ python_package_registry:
         - AWS CodeArtifact
     default: PyPI
 
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)